### PR TITLE
Get majority species

### PR DESCRIPTION
**Pull request by samwaseda**
Number of atoms of the majority species was wrong. And there was no DocString

### CODE REVIEW

**Comment by samwaseda** [comment]
I followed the order "number of atoms" then "species" but I must say it's more intuitive to have it other way around...

**Comment by samwaseda** [comment]
Actually even better would be to have `return_count` as an option and per default it returns only the majority species.

File: pyiron/atomistics/structure/atoms.py [hunk 1]
```diff
@@ -2201,15 +2201,17 @@ def get_equivalent_atoms(self, eps=1e-5):
...
 
         """
         el_dict = self.get_number_species_atoms()
         el_num = list(el_dict.values())
         el_name = list(el_dict.keys())
-        max_index = np.argsort(el_num)[-1]
-        return max_index, el_name[max_index]
+        if np.sum(np.array(el_num)==np.max(el_num))>1:
+            warnings.warn('There are more than one majority species')
+        return int(np.max(el_num)), el_name[np.argmax(el_num)]
```
Comment (sudarsan-surendralal) [inline pyiron/atomistics/structure/atoms.py]: Wouldn't it be more intuitive to return the species first and the count second?

Comment (sudarsan-surendralal) [inline pyiron/atomistics/structure/atoms.py]: Wouldn't it be more intuitive to return the species first and the count second?

File: tests/atomistics/structure/test_atoms.py [hunk 2]
```diff
@@ -666,6 +670,17 @@ def test_occupy_lattice(self):
         orig_basis.occupy_lattice(H=O_indices[0])
         self.assertEqual(orig_basis.get_chemical_formula(), "Cl31HNa32")
 
+
+    def test_get_majority_species(self):
+        basis = Atoms(symbols=4*['Fe'], positions=np.random.random((4, 3)), cell=np.eye(3))
```
Comment (sudarsan-surendralal) [inline tests/atomistics/structure/test_atoms.py]: Btw you can also initialize like

`Atoms(symbols='Fe4', positions=np.random.random((4, 3)), cell=np.eye(3))`

File: pyiron/atomistics/structure/atoms.py [hunk 3]
```diff
@@ -2201,15 +2201,17 @@ def get_equivalent_atoms(self, eps=1e-5):
...
+
+        Returns: number of atoms of the majority species, chemical symbol
 
         """
         el_dict = self.get_number_species_atoms()
         el_num = list(el_dict.values())
         el_name = list(el_dict.keys())
-        max_index = np.argsort(el_num)[-1]
-        return max_index, el_name[max_index]
+        if np.sum(np.array(el_num)==np.max(el_num))>1:
```
Comment (sudarsan-surendralal) [inline pyiron/atomistics/structure/atoms.py]: pep8 `if np.sum(np.array(el_num)==np.max(el_num)) > 1:`

File: pyiron/atomistics/structure/atoms.py [hunk 4]
```diff
@@ -2201,15 +2201,17 @@ def get_equivalent_atoms(self, eps=1e-5):
 
     def get_majority_species(self):
         """
-        
-        Returns:
+        This function returns the majority species and their number in the box
+
+        Returns: number of atoms of the majority species, chemical symbol
```
Comment (sudarsan-surendralal) [inline pyiron/atomistics/structure/atoms.py]: Enter and tab after Returns

``` 
Returns: 
       list: number of atoms of the majority species, chemical symbol
```

File: pyiron/atomistics/structure/atoms.py (hunk 1 above)
Comment (samwaseda) [reply pyiron/atomistics/structure/atoms.py]: I totally agree (cf. my comment above)

**Pull request merged**